README.md

```diff
@@ -112,19 +112,20 @@ whether the resource should be included in the filtered results.
 There are a few built-in predicates and some helper functions for
 creating your own:
 
-* `All` effectively AND's its arguments together
-* `Any` OR's its arguments together
-* `None` negates its arguments, returning false if any return true
+* `And` effectively AND's its arguments together (you must provide >=2
```

Comment:
this doc fallout is a bit funny :)

Reply:
Haha. There's probably a clearer way to put it, but I can deal with that later.